### PR TITLE
Add V7_TAG_STRING_6

### PR DESCRIPTION
**Pull request by cpq**
No coverage changes.
Inlines strings of length 6.

### CODE REVIEW

**Comment by mmikulicic** [comment]
Let's see how the discussion in #327 about zero termination goes.
It might be necessary to use a special tag to encode STRING_5 and use the space taken by the string length for storing a null byte.

**Comment by cpq** [comment]
I like zero-termination idea. Let's do it as a separate effort.

**Comment by mmikulicic** [comment]
Ok, let's abandon this, and keep the branch since we'll likely do a similar thing for 5-strings

**Comment by cpq** [comment]
Why not merging?
Adding 0-termination is an unrelated effort.
Regardless of it, adding an extra tag to hold a string that exactly fits NAN payload is a good idea.

**Comment by mmikulicic** [comment]
The problem is that if it fits exactly the NaN payload it cannot be null terminated.

So basically this tag will be used for fitting a 5-string plus it's null terminating byte.
Here, I'm assuming that we want to have all strings to be null terminated, that inline strings can have null bytes in them (hence the length will still be encoded in the first byte); but I'd defer to a design discussion to see if those assumptions make sense.

Basically, there are a few choices to be made, and I thought that making them without the burden of caring for another string type would be leaner.
